test/integration/util/util.go

```diff
@@ -331,14 +348,13 @@ func UpdateNodeStatus(cs clientset.Interface, node *v1.Node) error {
 
 // InitTestAPIServer initializes a test environment and creates an API server with default
 // configuration.
+// It registers cleanup functions to t.Cleanup(), they will be called when the test completes,
+// no need to do this again.
 func InitTestAPIServer(t *testing.T, nsPrefix string, admission admission.Interface) *TestContext {
-	ctx, cancelFunc := context.WithCancel(context.Background())
-	testCtx := TestContext{
-		Ctx:      ctx,
-		CancelFn: cancelFunc,
-	}
+	ctx, cancel := context.WithCancel(context.Background())
```

Comment:
I think a `ktesting.NewTestContext` is missing here.

Reply:
Yes, thanks for catching this. updated.